lib/audited/auditor.rb

```diff
@@ -217,7 +217,8 @@ def write_audit(attrs)
       def require_comment
         if auditing_enabled && audit_comment.blank?
           errors.add(:audit_comment, "Comment required before destruction")
-          return false
+          return false if Rails.version.start_with?('4.')
+          throw :abort
```

Comment:
See [this comment](https://github.com/collectiveidea/audited/pull/252#issuecomment-179853559) to the PR.